src/components/buttons-group.js

```diff
@@ -1,7 +1,7 @@
-function buttonsGroup(divsId,buttons,htmlClass) {		
+function buttonsGroup(groupsId,buttons,htmlClass) {		
 	return `
-		<div id="${divsId}" class="${htmlClass}">
+		<section id="${groupsId}" class="${htmlClass}">
```

Comment:
It's not needed

Reply:
no make any sense use section inside of button group, it's not a `section`

my last comment was just an example, in case you change div to other that like `p`, `span`, and so on!

in this case doesn't make sense you receive `divsId`, because it could change to other thing that isn't a div!